docs/docs/usage/scaladoc/settings.md

```diff
@@ -0,0 +1,113 @@
+
+## Overview of all available settings
+
+##### -project
+The name of the project. To provide compatibility with Scala2 aliases with `-doc-title`
+
+##### -project-version
+The current version of your project that appears in a top left corner. To provide compatibility with Scala2 aliases with `-doc-version`
+
+##### -project-logo
+The logo of your project that appears in a top left corner. To provide compatibility with Scala2 aliases with `-doc-logo`
+
+##### -project-footer
+The string message that appears in a footer section. To provide compatibility with Scala2 aliases with `-doc-footer`
+
+##### -comment-syntax
+The default syntax of comments. Defaults to markdown however wikisyntax can be used.
+
+##### -revision
+Revision (branch or ref) used to build project project. Useful with sourcelinks to prevent them from pointing always to the newest master that is subject to changes.
+
+##### -source-links
+Source links provide a mapping between file in documentation and code repository.
+
+Accepted formats:
+\<sub-path>=\<source-link>
+\<source-link>
+
+where <source-link> is one of following:
+ - `github://<organization>/<repository>[/revision][#subpath]`
+     will match https://github.com/$organization/$repository/\[blob|edit]/$revision\[/$subpath]/$filePath\[$lineNumber]
+     when revision is not provided then requires revision to be specified as argument for scaladoc
+ - `gitlab://<organization>/<repository>`
+     will match https://gitlab.com/$organization/$repository/-/\[blob|edit]/$revision\[/$subpath]/$filePath\[$lineNumber]
+     when revision is not provided then requires revision to be specified as argument for scaladoc
+ - \<scaladoc-template>
+
+<scaladoc-template> is a format for `doc-source-url` parameter scaladoc.
+NOTE: We only supports `€{FILE_PATH_EXT}`, `€{TPL_NAME}`, `€{FILE_EXT}`,
+ €{FILE_PATH}, and €{FILE_LINE} patterns
+
+
+Template can defined only by subset of sources defined by path prefix represented by `<sub-path>`.
+In such case paths used in templates will be relativized against `<sub-path>`
+
+Example source links is:
+`-source-links:docs=github://lampepfl/dotty/master#docs`
+
+
+##### -external-mappings
+
+Mapping between regexes matching classpath entries and external documentation.
+'regex::\[scaladoc|scaladoc|javadoc]::path' syntax is used
+
+Example external mapping is:
+`-external-mappings:.*scala.*::scaladoc3::http://dotty.epfl.ch/api/,.*java.*::javadoc::https://docs.oracle.com/javase/8/docs/api/`
+
+##### -social-links
+
+Links to social sites. '[github|twitter|gitter|discord]::link' syntax is used. 'custom::link::white_icon_name::black_icon_name' is also allowed, in this case icons must be present in 'images/'' directory.
+
+Example social link is:
+
+`-social-links:github::https://github.com/lampepfl/dotty,gitter::https://gitter.im/scala/scala,twitter::https://twitter.com/scala_lang`
```

Comment:
Again, I suggest putting the example first:

```suggestion
Links to social sites. For example:

`-social-links:github::https://github.com/lampepfl/dotty,gitter::https://gitter.im/scala/scala,twitter::https://twitter.com/scala_lang`

Valid values are of the form: '\[github|twitter|gitter|discord]::link'. Scaladoc also supports 'custom::link::white_icon_name::black_icon_name'. In this case icons must be present in 'images/'' directory.
```

Reply:
Fixed